src/pages/dmt5/_start-trader.js

```diff
@@ -0,0 +1,292 @@
+        }
+    }
+`
+const TabItem = styled.div`
+    padding: 2.4rem 4rem;
+    max-width: 30.8rem;
+    width: 100%;
+    height: 8.4rem;
+    border-radius: 4px;
+    border: solid 1px rgba(51, 51, 51, 0.1);
+    cursor: pointer;
+    ${(props) =>
+        props.active
+            ? css`
+                  box-shadow: 0 16px 20px 0 rgba(0, 0, 0, 0.05), 0 0 20px 0 rgba(0, 0, 0, 0.05);
+                  border: unset;
+                  ${Text} {
+                      font-weight: bold;
+                  }
+              `
+            : css`
+                  box-shadow: unset;
+                  ${Text} {
+                      font-weight: unset;
+                  }
+              `}
+
+    @media ${device.mobileL} {
+        padding: 24px 24px;
+
+        :last-child{
+            padding: 12px 24px;
+        }
+    }
+`
+const Span = styled.span`
+    color: red;
+`
+
+const StyledHeader = styled(Header)`
+    @media ${device.mobileL} {
+        font-size: 32px;
+        margin-bottom: 24px;
+        padding: 0 16px;
+    }
+`
+const StyledText = styled(Text)`
+    @media ${device.mobileL} {
+        font-size: 16px;
+    }
+`
+
+const StartTrader = () => {
+    const [is_mobile, setMobile] = useState(false)
+    const handleResizeWindow = () => {
+        setMobile(isBrowser() ? window.screen.width <= size.tabletS : false)
+    }
+    useEffect(() => {
+        setMobile(isBrowser() ? window.screen.width <= size.tabletS : false)
+        window.addEventListener('resize', handleResizeWindow)
+    })
+
+    const data = useStaticQuery(query)
+    const [tab, setTab] = useState('Demo')
+
+    const onTabClick = (tab) => {
+        setTab(tab)
+    }
+    return (
+        <Section>
+            <StyledHeader align="center" mb="4rem" as="h2">
+                {localize('How to get started with a DMT5 account')}
+            </StyledHeader>
+            <Flex mb="8rem" p="0 16px" tablet={{ mb: '32px', height: 'unset' }}>
+                <TabItem active={tab === 'Demo'} onClick={() => onTabClick('Demo')}>
+                    <StyledText size="var(--text-size-m)" align="center">
+                        {localize('Demo account')}
+                    </StyledText>
+                </TabItem>
+                <TabItem active={tab === 'Real'} onClick={() => onTabClick('Real')}>
+                    <StyledText size="var(--text-size-m)" align="center">
+                        {localize('Real money account')}
+                    </StyledText>
+                </TabItem>
+            </Flex>
+
+            <Flex max_width="1200px">
+                {tab === 'Demo' ? (
+                    <SideTab>
+                        <SideTab.Panel
+                            label=""
+                            description={
+                                <Localize
+                                    translate_text="Sign up for a free <0>Deriv demo account</0>"
+                                    components={[<Span color="red" key={0} />]}
```

Comment:
I suppose the link to signup page should be added here?

Reply:
Thank you for mentioning. I will ask designers to get the confirmation.